client/errors.go

```diff
@@ -6,23 +6,38 @@ import (
 )
 
 var (
-	ErrMissingServer      = errors.New("server address is unset or empty")
+	// ErrMissingServer is returned when server endpoint is empty in parameters.
+	ErrMissingServer = errors.New("server address is unset or empty")
+	// ErrNonPositiveTimeout is returned when any timeout is below zero in parameters.
 	ErrNonPositiveTimeout = errors.New("non-positive timeout")
 
-	ErrMissingContainer     = errors.New("missing container")
-	ErrMissingObject        = errors.New("missing object")
-	ErrMissingAccount       = errors.New("missing account")
-	ErrMissingEACL          = errors.New("missing eACL table")
+	// ErrMissingContainer is returned when container is not provided.
+	ErrMissingContainer = errors.New("missing container")
+	// ErrMissingObject is returned when object is not provided.
+	ErrMissingObject = errors.New("missing object")
+	// ErrMissingAccount is returned when account/owner is not provided.
+	ErrMissingAccount = errors.New("missing account")
+	// ErrMissingEACL is returned when eACL table is not provided.
+	ErrMissingEACL = errors.New("missing eACL table")
+	// ErrMissingEACLContainer is returned when container info is not provided in eACL table.
 	ErrMissingEACLContainer = errors.New("missing container in eACL table")
+	// ErrMissingAnnouncements is returned when announcements are not provided.
 	ErrMissingAnnouncements = errors.New("missing announcements")
-	ErrZeroRangeLength      = errors.New("zero range length")
-	ErrMissingRanges        = errors.New("missing ranges")
-	ErrZeroEpoch            = errors.New("zero epoch")
-	ErrMissingTrusts        = errors.New("missing trusts")
-	ErrMissingTrust         = errors.New("missing trust")
-
+	// ErrZeroRangeLength is returned when range parameter has zero length.
+	ErrZeroRangeLength = errors.New("zero range length")
+	// ErrMissingRanges is returned when empty ranges list is provided.
+	ErrMissingRanges = errors.New("missing ranges")
+	// ErrZeroEpoch is returned when zero epoch is provided.
+	ErrZeroEpoch = errors.New("zero epoch")
+	// ErrMissingTrusts is returned when empty slice of trusts is provided.
+	ErrMissingTrusts = errors.New("missing trusts")
+	// ErrMissingTrust is returned when empty trust is not provided.
+	ErrMissingTrust = errors.New("missing trust")
+
+	// ErrUnexpectedReadCall is returned when we already got all data but truing to get more.
 	ErrUnexpectedReadCall = errors.New("unexpected call to `Read`")
 
+	// ErrSign is returned when unable to sign service message.
```

Comment:
You better add comments along with new vars.

Reply:
Agree, working on this